src/kaocha/plugin/alpha/spec_check.clj

```diff
@@ -0,0 +1,41 @@
+(ns kaocha.plugin.alpha.spec-check
+  (:require [clojure.spec.alpha :as s]
+            [kaocha.hierarchy :as kaocha]
+            [kaocha.plugin :refer [defplugin]]
+            [kaocha.testable :as testable]
+            [kaocha.type.clojure.test]
+            [kaocha.type :as type]
+            [kaocha.type.clojure.spec.alpha.check]
+            [kaocha.type.clojure.spec.alpha.fdef :as type.fdef]))
+
+(alias 'stc 'clojure.spec.test.check)
+(alias 'type.stc 'kaocha.type.clojure.spec.alpha.check)
+
+(defn testable [{ns-patterns ::ns-patterns :as config}]
+  (-> {:kaocha.testable/type    :kaocha.type/clojure.spec.test.alpha.check
+       :kaocha.testable/id      :generative
+       :kaocha/ns-patterns      ns-patterns
+       :kaocha/source-paths     ["src"],
+       :kaocha.filter/skip-meta [:kaocha/skip]
+       ::type.stc/syms          :all-fdefs}
+      (merge (type.fdef/stc-opts config))))
+
+(defplugin kaocha.plugin.alpha/spec-check
+  (pre-load [config] (update config :kaocha/tests conj (testable config)))
```

Comment:
It seems the idea here is that people invoke these tests through the plugin? i.e. instead of having an entry in `tests.edn` they enable the plugin?

I'd rather turn it around. I've been thinking for a while that test types should be allowed to register plugins, so you can have this plugin be automatically active when someone adds a test suite of type `:kaocha/clojure.spec.test.alpha.check`, so that people can then use the `--spec-check-num-tests` and `--spec-check-max-size`command line flags.

Reply:
My goal is just to allow people to add some simplistic spec checking purely through the CLI. But as I have it written, you could also add things to `tests.edn` if you need more fine-grained control, per @RickMoynihan's proposal.

I personally don't like maintaining `tests.edn`s since I have quite a few projects, so I wanted a way to do this through just CLI. That's the only purpose of the plugin. Is there a better way to offer that functionality?

Reply:
Isn't it easier to add 

```
#kaocha/v1
{:tests [{:type :kaocha.type/clojure.spec.test.alpha.check}]}
```

to the project once, rather than adding `--plugin kaocha.plugin.alpha/spec-check` to each invocation?

Reply:
Not in my case, since I have a lot of projects, and I can just CTL+R to recall my previous test run invocation for any given project. (Or put it in a binstub somewhere on my system). That's my preferred workflow.

Reply:
Er I guess since we have to specify ns-patterns it would be a different invocation per project, but it's still not necessarily something I'd want to manage in source control for each individual project. That may change, though, if some projects have really specific tuning requirements. But I like to have the option of CLI-only config.

Reply:
(furthermore, I'd love to not require ns-patterns. I'd much prefer if we could figure that out automatically. I'll look into that.)

Reply:
This does go against Kaocha's philosophy, one of the core ideas is that your test configuration lives with the project so that you can use the same invocation (`bin/kaocha`) everywhere.

Reply:
Maybe when I get to a place where my kaocha workflow is more stable, I could do that. But right now it would be hard to maintain across all the projects while I figure out the workflow I want. I started out using `tests.edn` but ran into this problem, so I switched to CLI arguments. I love that your project supports both. 

Also, right now I'm the only developer on these projects, but when (if) other developers join me, it strikes me as something that each developer might want to run a little differently (different reporting preferences, different performance requirements, etc.). So they'd possibly want to have their own version of `tests.edn`, in which case it wouldn't be source controlled.